src/jquery.multiselect.js

```diff
@@ -29,18 +29,23 @@
       height: 175,
       minWidth: 225,
       classes: '',
-      checkAllText: 'Check all',
-      uncheckAllText: 'Uncheck all',
-      noneSelectedText: 'Select options',
+      openIcon: '<span class="ui-icon ui-icon-triangle-1-s"></span>',   // Scaleable HTML Entities or Font-Awesome icons can be used here instead of the default jQuery UI icons.
+      closeIcon: '<span class="ui-icon ui-icon-circle-close"></span>',
+      checkAllIcon: '<span class="ui-icon ui-icon-check"></span>',
+      uncheckAllIcon: '<span class="ui-icon ui-icon-closethick"></span>',
+      flipAllIcon: '<span class="ui-icon ui-icon-arrowrefresh-1-w"></span>',
+      checkAllText: 'Check all',																							// If blank or null, link not shown.
```

Comment:
It looks like there's a bunch of white space here to try and line these comments up. I'm guessing tabs? It's not consistently lined up on my editor, probably a tab width thing. I'd prefer you just stick with spaces and put them next to the options.

Reply:
Yeah, I am fighting with the tab vs space thing.   Generally, I prefer tabs, but I realize I need spaces here.